### ISSUE TITLE
Fairer Danger List

### ISSUE THREAD

**Issue by Cerno-b** [ludumdare/ludumdare#967]
I would like to suggest a modification to the Danger category.

I find it almost offensive that a game with Ratings Received / Ratings given (18/3) can be on the front page at position 2 of the Danger List while a game with 13/28 is pushed down to page 2.

Maybe the Danger list can be sorted **by Received \* Coolness** instead of Received (as I assume it’s currently done). This change would favor games that contributed and are close to the minimum votes over of those that didn’t contribute.

Another option would be to have an additional filter “below 20 received”. That way people could simply use the coolness filter, switch on the “below 20” filter and get all deserving games, not only those that are easy to save.

**Comment by pfefferminzz** [comment]
Games with Ratings given: 0 should never be able to be on the first Danger page. A simple approach would be:

Sort the list as you like (Classic, Smart, Karma), but at the end it is divided in two chunks:
_All games with Ratings Given > Ratings Recieved are shown before all games with RG <= RR at all times._

**Comment by pfefferminzz** [comment]
I agree. The Danger filter currently shows the games, which are easy to save, regardless of their effort to give ratings to other games.

**Comment by local-minimum** [comment]
@Cerno-b can we merge this issue with the other identical perhaps and put your suggestion in the comment there to make it less confusing?

**Comment by Cerno-b** [comment]
Definitely.

**Comment by povrazor** [comment]
Long story short, `Smart` is basically `Received * Coolness`, but a version that caps at <20 could help.

**Comment by zwrawr** [comment]
Also some talk here #971
